


<!DOCTYPE html>
<html lang="ru">

<!-- Mirrored from archlinux.org.ru/forum/topic/21119/ by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Sep 2022 04:11:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>
        Перестал работать интернет в Kodi
    </title>
    <link rel="shortcut icon" href="https://storage.yandexcloud.net/aor-static/static/favicon.ico" type="image/x-icon" />
    <link rel="stylesheet" type="text/css" href="https://storage.yandexcloud.net/aor-static/static/css/default.css">
    
    
    

    <script type="text/javascript" src="https://storage.yandexcloud.net/aor-static/static/jquery-1.8.3.min.js"></script>
    <script type="text/javascript" src="https://storage.yandexcloud.net/aor-static/static/js/bootstrap.min.js"></script>
    <script type="text/javascript" src="https://storage.yandexcloud.net/aor-static/static/pybb/js/pybbjs.js"></script>
    
    
    
    <!-- Feeds -->
    <link rel="alternate" type="application/atom+xml" href="../../feeds/posts/index.html" title="Последние сообщения на форуме" >
    <link rel="alternate" type="application/atom+xml" href="../../feeds/topics/index.html" title="Последние темы на форуме" >

    
    
    
    <script type="text/javascript" src="https://storage.yandexcloud.net/aor-static/static/pybb/js/jquery.formset.min.js"></script>

</head>
<body>

<nav class="navbar navbar-default" role="navigation">
    <div class="container-fluid wrapper">
        
    
<ul class="nav navbar-nav">
    <li>
      <a class="navbar-brand" href="../../../index.html">
        <img src="https://storage.yandexcloud.net/aor-static/static/logo.png" alt="" style="opacity: .7; vertical-align: top">
      </a>
    </li>
    <li class=""><a href="../../../news/index.html">Новости</a></li>
    <li class="active"><a href="../../index.html">Форум</a></li>
    <li class=""><a href="../../../blogs/index.html">Блоги</a></li>
    <li><a href="https://archlinux.org/">Eng</a></li>
    <li><a href="https://wiki.archlinux.org/index.php/Main_page_(Русский)">Wiki</a></li>
    <li class="dropdown">
      <a href="https://www.archlinux.org/packages/" class="dropdown-toggle" data-toggle="dropdown">
        Пакеты
        <b class="caret"></b>
      </a>
      <ul class="dropdown-menu">
        <li><a href="https://www.archlinux.org/packages/">Официальные репозитории</a></li></li>
        <li><a href="https://aur.archlinux.org/?setlang=ru">AUR</a></li>
      </ul>
    <li><a href="https://archlinux.org/download/">Скачать</a></li>
</ul>

<form class="navbar-form navbar-left" role="search" action="https://www.google.ru/cse">
    <input type="hidden" name="cx" value="018428664132160064944:ntsma9ch2ug"/>
    <input type="hidden" name="ie" value="UTF-8"/>
    <div class="input-group">
      <input type="text" name="q" class="form-control" placeholder="Что ищем?">
      <span class="input-group-btn">
        <button type="submit" name="sa" class="btn btn-default">Найти</button>
      </span>
    </div>
</form>

<ul class="nav navbar-nav navbar-right">


    
      <li><a href="../../../irc/index.html#chat">IRC</a></li>
      <li class="dropdown">
        <a href="../../../accounts/login/index.html" class="dropdown-toggle" data-toggle="dropdown">
          Профиль
          <b class="caret"></b>
        </a>
        <ul class="dropdown-menu">
          <li><a href="../../../accounts/login/index.html">Вход</a></li>
          <li><a href="../../../accounts/register/index.html">Регистрация</a></li>
        </ul>
      </li>
    
</ul>


    </div>
</nav>



<div class="modal fade" id='new_pm_notify' role="dialog" aria-labelledby="gridSystemModalLabel">
  <div class="modal-dialog modal-sm" role="document">
    <div class="modal-content navbar-default">
      <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
        <h4 class="modal-title" id="gridSystemModalLabel">Новые личные сообщения</h4>
      </div>
      <div class="modal-body">
        <a href="../../../messages/inbox/index.html" class="center-block btn btn-primary">Перейти в папку Входящие <span class="badge"></span></a>
      </div>
    </div>
  </div>
</div>

<div class="container-fluid wrapper">
    <div class="row">
        <div class="col-md-12">
            
    
        
<ul class="breadcrumb">
    

    <li><a href="../../../index.html">ArchLinux.org.ru</a></li>


    <li><a href="../../index.html">Начало</a> <span class="divider">/</span></li>
    
        
            
                <li><a href="../../category/2/index.html">Поддержка пользователей</a> <span class="divider">/</span></li>
            
                <li><a href="../../forum/9/index.html">Новичкам в Arch Linux</a> <span class="divider">/</span></li>
            
        
        
            <li>Перестал работать интернет в Kodi</li>
        
    
    
</ul>

    

        </div>
    </div>
    <div class="row">
        <div class="col-md-12">
            
    <div class="row">
        <div class="col-md-12 forum">
            
    
    <div class="topic">
        <h1>Перестал работать интернет в Kodi</h1>
        
            <div class="text-center">
    



</div>

        

        

        <div class="posts">
            
            
                
                




<table class="table table-bordered post post-row
       "
       id="post-253797"
       >
    <thead>
    <tr>
        <td class="post-author">
            <a href="../../users/abc/index.html">
                <span class="post-username">abc</span>
            </a>
        </td>
        <td class="post-date">
            <div class="post-control">
                
                    
                
            </div>
            <a class="permalink" href="../../post/253797/index.html">#</a>
            8 месяцев, 1 неделя назад
            
            
        </td>
    </tr>
    </thead>
    <tbody>
    <tr>
        <td class="post-info">
            

<div class="avatar">
    
</div>

            <div>
                Темы:
                <a href="../../users/abc/topics/index.html">
                    38
                </a>
            </div>
            <div>
                Сообщения:
                <a href="../../users/abc/posts/index.html">
                    223
                </a>
            </div>
            <div>
                Участник с: 30 августа 2016
            </div>
            
        </td>
        <td class="post-content">
            Около месяца назад установил Коди. Все настроил, все работало. Несколько дней назад перестала обновляться погода через плагин Гисметео. Так как он мне не критично нужен, не придал этому значения, подумал что в плагине что-то сломалось. А вчера отвалился плагин для Ютуба - ни одно видео не показывает. Так же не показывает с ютуба, если отправлять ссылку с телефона. Следом перестал работать плагин Яндекс музыки.<br /><br />В настройках Коди показывает, что интернет есть. Айпт от роутера получен.<br /><br />В терминале пинг идёт, wget и yt-dlp скачивают.<br /><br />Что могло сломаться в коди?
            
                
            

            <div class="post-related">
                <div class='attachments'>
                    
                </div>
            </div>
        </td>
    </tr>
    </tbody>
</table>

            
                
                




<table class="table table-bordered post post-row
       "
       id="post-253798"
       >
    <thead>
    <tr>
        <td class="post-author">
            <a href="../../users/vs220/index.html">
                <span class="post-username">vs220</span>
            </a>
        </td>
        <td class="post-date">
            <div class="post-control">
                
                    
                
            </div>
            <a class="permalink" href="../../post/253798/index.html">#</a>
            8 месяцев, 1 неделя назад
            
                (отредактировано
                
                    8 месяцев, 1 неделя назад)
                
            
            
        </td>
    </tr>
    </thead>
    <tbody>
    <tr>
        <td class="post-info">
            

<div class="avatar">
    
</div>

            <div>
                Темы:
                <a href="../../users/vs220/topics/index.html">
                    22
                </a>
            </div>
            <div>
                Сообщения:
                <a href="../../users/vs220/posts/index.html">
                    8070
                </a>
            </div>
            <div>
                Участник с: 16 августа 2009
            </div>
            
        </td>
        <td class="post-content">
            Лог ?<br /><a href="https://kodi.wiki/view/Log_file/Advanced">https://kodi.wiki/view/Log_file/Advanced</a><br /><br /><pre><code>cat  ~/.kodi/temp/kodi.log</code></pre>
            
                
            

            <div class="post-related">
                <div class='attachments'>
                    
                </div>
            </div>
        </td>
    </tr>
    </tbody>
</table>

            
                
                




<table class="table table-bordered post post-row
       "
       id="post-253804"
       >
    <thead>
    <tr>
        <td class="post-author">
            <a href="../../users/abc/index.html">
                <span class="post-username">abc</span>
            </a>
        </td>
        <td class="post-date">
            <div class="post-control">
                
                    
                
            </div>
            <a class="permalink" href="../../post/253804/index.html">#</a>
            8 месяцев, 1 неделя назад
            
            
        </td>
    </tr>
    </thead>
    <tbody>
    <tr>
        <td class="post-info">
            

<div class="avatar">
    
</div>

            <div>
                Темы:
                <a href="../../users/abc/topics/index.html">
                    38
                </a>
            </div>
            <div>
                Сообщения:
                <a href="../../users/abc/posts/index.html">
                    223
                </a>
            </div>
            <div>
                Участник с: 30 августа 2016
            </div>
            
        </td>
        <td class="post-content">
            <blockquote><em>vs220</em><br>Лог</blockquote>Как раз оттуда и начал смотреть. Но последняя запись в этот файл была 2021-11-27.<br /><br />Нашел в /var/lib/kodi/.kodi/temp. Не знаю то ли это?<br /><pre><code>
2021-12-30 06:42:16.594 T:14824    INFO &lt;general&gt;: Python interpreter stopped
2021-12-30 07:12:16.617 T:15010    INFO &lt;general&gt;: WEATHER: Downloading weather
2021-12-30 07:12:16.618 T:15011    INFO &lt;general&gt;: initializing python engine.
2021-12-30 07:12:16.782 T:15011   ERROR &lt;general&gt;: EXCEPTION Thrown (PythonToCppException) : --&gt;Python callback/script returned the following error&lt;--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: &lt;class &#39;ImportError&#39;&gt;
                                                   Error Contents: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   Traceback (most recent call last):
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/default.py&quot;, line 12, in &lt;module&gt;
                                                       from resources.libs import Gismeteo, GismeteoError, Location, Weather, WebClientError
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/__init__.py&quot;, line 9, in &lt;module&gt;
                                                       from .simpleweather import Weather as SW_Weather
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/__init__.py&quot;, line 6, in &lt;module&gt;
                                                       from .simpleweather import (WeatherProperties, Weather,
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/simpleweather.py&quot;, line 9, in &lt;module&gt;
                                                       import simpleplugin
                                                     File &quot;/var/lib/kodi/.kodi/addons/script.module.simpleplugin3/libs/simpleplugin.py&quot;, line 30, in &lt;module&gt;
                                                       from collections import MutableMapping, namedtuple
                                                   ImportError: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   --&gt;End of Python script error report&lt;--

2021-12-30 07:12:16.805 T:15011    INFO &lt;general&gt;: Python interpreter stopped
2021-12-30 07:42:16.823 T:15175    INFO &lt;general&gt;: WEATHER: Downloading weather
2021-12-30 07:42:16.823 T:15176    INFO &lt;general&gt;: initializing python engine.
2021-12-30 07:42:16.992 T:15176   ERROR &lt;general&gt;: EXCEPTION Thrown (PythonToCppException) : --&gt;Python callback/script returned the following error&lt;--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: &lt;class &#39;ImportError&#39;&gt;
                                                   Error Contents: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   Traceback (most recent call last):
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/default.py&quot;, line 12, in &lt;module&gt;
                                                       from resources.libs import Gismeteo, GismeteoError, Location, Weather, WebClientError
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/__init__.py&quot;, line 9, in &lt;module&gt;
                                                       from .simpleweather import Weather as SW_Weather
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/__init__.py&quot;, line 6, in &lt;module&gt;
                                                       from .simpleweather import (WeatherProperties, Weather,
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/simpleweather.py&quot;, line 9, in &lt;module&gt;
                                                       import simpleplugin
                                                     File &quot;/var/lib/kodi/.kodi/addons/script.module.simpleplugin3/libs/simpleplugin.py&quot;, line 30, in &lt;module&gt;
                                                       from collections import MutableMapping, namedtuple
                                                   ImportError: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   --&gt;End of Python script error report&lt;--

2021-12-30 07:42:17.016 T:15176    INFO &lt;general&gt;: Python interpreter stopped
2021-12-30 08:12:17.035 T:15369    INFO &lt;general&gt;: WEATHER: Downloading weather
2021-12-30 08:12:17.035 T:15370    INFO &lt;general&gt;: initializing python engine.
2021-12-30 08:12:17.195 T:15370   ERROR &lt;general&gt;: EXCEPTION Thrown (PythonToCppException) : --&gt;Python callback/script returned the following error&lt;--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: &lt;class &#39;ImportError&#39;&gt;
                                                   Error Contents: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   Traceback (most recent call last):
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/default.py&quot;, line 12, in &lt;module&gt;
                                                       from resources.libs import Gismeteo, GismeteoError, Location, Weather, WebClientError
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/__init__.py&quot;, line 9, in &lt;module&gt;
                                                       from .simpleweather import Weather as SW_Weather
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/__init__.py&quot;, line 6, in &lt;module&gt;
                                                       from .simpleweather import (WeatherProperties, Weather,
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/simpleweather.py&quot;, line 9, in &lt;module&gt;
                                                       import simpleplugin
                                                     File &quot;/var/lib/kodi/.kodi/addons/script.module.simpleplugin3/libs/simpleplugin.py&quot;, line 30, in &lt;module&gt;
                                                       from collections import MutableMapping, namedtuple
                                                   ImportError: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   --&gt;End of Python script error report&lt;--

2021-12-30 08:12:17.217 T:15370    INFO &lt;general&gt;: Python interpreter stopped
2021-12-30 08:42:17.241 T:15547    INFO &lt;general&gt;: WEATHER: Downloading weather
2021-12-30 08:42:17.241 T:15548    INFO &lt;general&gt;: initializing python engine.
2021-12-30 08:42:17.399 T:15548   ERROR &lt;general&gt;: EXCEPTION Thrown (PythonToCppException) : --&gt;Python callback/script returned the following error&lt;--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: &lt;class &#39;ImportError&#39;&gt;
                                                   Error Contents: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   Traceback (most recent call last):
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/default.py&quot;, line 12, in &lt;module&gt;
                                                       from resources.libs import Gismeteo, GismeteoError, Location, Weather, WebClientError
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/__init__.py&quot;, line 9, in &lt;module&gt;
                                                       from .simpleweather import Weather as SW_Weather
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/__init__.py&quot;, line 6, in &lt;module&gt;
                                                       from .simpleweather import (WeatherProperties, Weather,
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/simpleweather.py&quot;, line 9, in &lt;module&gt;
                                                       import simpleplugin
                                                     File &quot;/var/lib/kodi/.kodi/addons/script.module.simpleplugin3/libs/simpleplugin.py&quot;, line 30, in &lt;module&gt;
                                                       from collections import MutableMapping, namedtuple
                                                   ImportError: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   --&gt;End of Python script error report&lt;--

2021-12-30 08:42:17.421 T:15548    INFO &lt;general&gt;: Python interpreter stopped
2021-12-30 09:12:17.454 T:15748    INFO &lt;general&gt;: WEATHER: Downloading weather
2021-12-30 09:12:17.454 T:15749    INFO &lt;general&gt;: initializing python engine.
2021-12-30 09:12:17.613 T:15749   ERROR &lt;general&gt;: EXCEPTION Thrown (PythonToCppException) : --&gt;Python callback/script returned the following error&lt;--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: &lt;class &#39;ImportError&#39;&gt;
                                                   Error Contents: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   Traceback (most recent call last):
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/default.py&quot;, line 12, in &lt;module&gt;
                                                       from resources.libs import Gismeteo, GismeteoError, Location, Weather, WebClientError
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/__init__.py&quot;, line 9, in &lt;module&gt;
                                                       from .simpleweather import Weather as SW_Weather
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/__init__.py&quot;, line 6, in &lt;module&gt;
                                                       from .simpleweather import (WeatherProperties, Weather,
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/simpleweather.py&quot;, line 9, in &lt;module&gt;
                                                       import simpleplugin
                                                     File &quot;/var/lib/kodi/.kodi/addons/script.module.simpleplugin3/libs/simpleplugin.py&quot;, line 30, in &lt;module&gt;
                                                       from collections import MutableMapping, namedtuple
                                                   ImportError: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   --&gt;End of Python script error report&lt;--

2021-12-30 09:12:17.635 T:15749    INFO &lt;general&gt;: Python interpreter stopped
2021-12-30 09:42:17.655 T:15938    INFO &lt;general&gt;: WEATHER: Downloading weather
2021-12-30 09:42:17.656 T:15939    INFO &lt;general&gt;: initializing python engine.
2021-12-30 09:42:17.823 T:15939   ERROR &lt;general&gt;: EXCEPTION Thrown (PythonToCppException) : --&gt;Python callback/script returned the following error&lt;--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: &lt;class &#39;ImportError&#39;&gt;
                                                   Error Contents: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   Traceback (most recent call last):
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/default.py&quot;, line 12, in &lt;module&gt;
                                                       from resources.libs import Gismeteo, GismeteoError, Location, Weather, WebClientError
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/__init__.py&quot;, line 9, in &lt;module&gt;
                                                       from .simpleweather import Weather as SW_Weather
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/__init__.py&quot;, line 6, in &lt;module&gt;
                                                       from .simpleweather import (WeatherProperties, Weather,
                                                     File &quot;/var/lib/kodi/.kodi/addons/weather.gismeteo/resources/libs/simpleweather/simpleweather.py&quot;, line 9, in &lt;module&gt;
                                                       import simpleplugin
                                                     File &quot;/var/lib/kodi/.kodi/addons/script.module.simpleplugin3/libs/simpleplugin.py&quot;, line 30, in &lt;module&gt;
                                                       from collections import MutableMapping, namedtuple
                                                   ImportError: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)
                                                   --&gt;End of Python script error report&lt;--

2021-12-30 09:42:17.847 T:15939    INFO &lt;general&gt;: Python interpreter stopped
2021-12-30 09:49:09.586 T:578      INFO &lt;general&gt;: JSONRPC Server: New connection added
2021-12-30 09:49:10.077 T:533     ERROR &lt;general&gt;: Unable to find plugin plugin.video.sendtokodi
2021-12-30 09:49:40.510 T:578      INFO &lt;general&gt;: JSONRPC Server: Disconnection detected
</code></pre><br />Еще в /var/lib/kodi/ есть несколько файлов kodi_crashlog_*. В них в основном повторяются такие строки:<br /><pre><code>2021-12-25 22:52:55.556 T:630     ERROR &lt;general&gt;: Unable to lookup host: &#39;<a href="http://www.msftncsi.com/">www.msftncsi.com</a>&#39;
2021-12-25 22:52:55.560 T:630     ERROR &lt;general&gt;: CCurlFile::Exists - Failed: Couldn&#39;t resolve host name(6) for <a href="http://www.msftncsi.com/ncsi.txt">http://www.msftncsi.com/ncsi.txt</a>
2021-12-25 22:52:55.595 T:630     ERROR &lt;general&gt;: Unable to lookup host: &#39;<a href="http://www.w3.org/">www.w3.org</a>&#39;
</code></pre>
            
                
            

            <div class="post-related">
                <div class='attachments'>
                    
                </div>
            </div>
        </td>
    </tr>
    </tbody>
</table>

            
                
                




<table class="table table-bordered post post-row
       "
       id="post-253805"
       >
    <thead>
    <tr>
        <td class="post-author">
            <a href="../../users/vs220/index.html">
                <span class="post-username">vs220</span>
            </a>
        </td>
        <td class="post-date">
            <div class="post-control">
                
                    
                
            </div>
            <a class="permalink" href="../../post/253805/index.html">#</a>
            8 месяцев, 1 неделя назад
            
                (отредактировано
                
                    8 месяцев, 1 неделя назад)
                
            
            
        </td>
    </tr>
    </thead>
    <tbody>
    <tr>
        <td class="post-info">
            

<div class="avatar">
    
</div>

            <div>
                Темы:
                <a href="../../users/vs220/topics/index.html">
                    22
                </a>
            </div>
            <div>
                Сообщения:
                <a href="../../users/vs220/posts/index.html">
                    8070
                </a>
            </div>
            <div>
                Участник с: 16 августа 2009
            </div>
            
        </td>
        <td class="post-content">
            <blockquote><em>abc</em><br>ImportError: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)</blockquote>С питоном разбираться, почему не может найти. Все питоновские пакеты из официальных реп?<br />Хотя возможно старый пакет или скрипт просто какой то<br /><blockquote>.<br /> DeprecationWarning: Using or importing the ABCs from &#39;collections&#39; instead of from &#39;collections.abc&#39; is deprecated since Python 3.3, and in 3.10 it will stop working</blockquote>Тогда или переделывать скрипт или откатывать все питоновские на 3.9
            
                
            

            <div class="post-related">
                <div class='attachments'>
                    
                </div>
            </div>
        </td>
    </tr>
    </tbody>
</table>

            
                
                




<table class="table table-bordered post post-row
       "
       id="post-253806"
       >
    <thead>
    <tr>
        <td class="post-author">
            <a href="../../users/vasek/index.html">
                <span class="post-username">vasek</span>
            </a>
        </td>
        <td class="post-date">
            <div class="post-control">
                
                    
                
            </div>
            <a class="permalink" href="../../post/253806/index.html">#</a>
            8 месяцев, 1 неделя назад
            
                (отредактировано
                
                    8 месяцев, 1 неделя назад)
                
            
            
        </td>
    </tr>
    </thead>
    <tbody>
    <tr>
        <td class="post-info">
            

<div class="avatar">
    
</div>

            <div>
                Темы:
                <a href="../../users/vasek/topics/index.html">
                    48
                </a>
            </div>
            <div>
                Сообщения:
                <a href="../../users/vasek/posts/index.html">
                    11320
                </a>
            </div>
            <div>
                Участник с: 17 февраля 2013
            </div>
            
        </td>
        <td class="post-content">
            <blockquote><em>abc</em><br>Error Contents: cannot import name &#39;MutableMapping&#39; from &#39;collections&#39; (/usr/lib/python3.10/collections/__init__.py)</blockquote>Погугли по этой фразе  ... и станет понятно ... <a href="https://question-it.com/questions/74887/cli-aws-s-python-390a1-oshibka-iz-importa-kollektsij-mutablemapping">первое, что попалось</a> (хотя ссылка старая, около года)<br /><br /><strong>PS</strong> - хотя посмотрел внимательнее, <u>вроде бы это не  то</u> .... <u>но гуглить всеравно нужно обязательно</u>
            
                
                    <div class="post-signature">
                        Ошибки не исчезают с опытом - они просто умнеют
                    </div>
                
            

            <div class="post-related">
                <div class='attachments'>
                    
                </div>
            </div>
        </td>
    </tr>
    </tbody>
</table>

            
                
                




<table class="table table-bordered post post-row
       "
       id="post-253807"
       >
    <thead>
    <tr>
        <td class="post-author">
            <a href="../../users/abc/index.html">
                <span class="post-username">abc</span>
            </a>
        </td>
        <td class="post-date">
            <div class="post-control">
                
                    
                
            </div>
            <a class="permalink" href="../../post/253807/index.html">#</a>
            8 месяцев, 1 неделя назад
            
            
        </td>
    </tr>
    </thead>
    <tbody>
    <tr>
        <td class="post-info">
            

<div class="avatar">
    
</div>

            <div>
                Темы:
                <a href="../../users/abc/topics/index.html">
                    38
                </a>
            </div>
            <div>
                Сообщения:
                <a href="../../users/abc/posts/index.html">
                    223
                </a>
            </div>
            <div>
                Участник с: 30 августа 2016
            </div>
            
        </td>
        <td class="post-content">
            <blockquote><em>vs220</em><br>Все питоновские пакеты из официальных реп?</blockquote>Отдельно ничего связанного с питоном не ставил.<br /><br />Сейчас у меня Python 3.10.1.<br /><br />Погуглил. Да, дело именно в версии питона. Будем разбираться. Спасибо за наводку.
            
                
            

            <div class="post-related">
                <div class='attachments'>
                    
                </div>
            </div>
        </td>
    </tr>
    </tbody>
</table>

            
                
                




<table class="table table-bordered post post-row
       "
       id="post-253814"
       >
    <thead>
    <tr>
        <td class="post-author">
            <a href="../../users/vs220/index.html">
                <span class="post-username">vs220</span>
            </a>
        </td>
        <td class="post-date">
            <div class="post-control">
                
                    
                
            </div>
            <a class="permalink" href="../../post/253814/index.html">#</a>
            8 месяцев, 1 неделя назад
            
                (отредактировано
                
                    8 месяцев, 1 неделя назад)
                
            
            
        </td>
    </tr>
    </thead>
    <tbody>
    <tr>
        <td class="post-info">
            

<div class="avatar">
    
</div>

            <div>
                Темы:
                <a href="../../users/vs220/topics/index.html">
                    22
                </a>
            </div>
            <div>
                Сообщения:
                <a href="../../users/vs220/posts/index.html">
                    8070
                </a>
            </div>
            <div>
                Участник с: 16 августа 2009
            </div>
            
        </td>
        <td class="post-content">
            <blockquote><em>abc</em><br>ничего связанного с питоном не ставил</blockquote><br />Скрипт <blockquote>/var/lib/kodi/.kodi/addons/weather.gismeteo/default.py</blockquote> не из официальных реп арча, с 3.10 не работает надо переделывать в соответствии с новыми правилами.<br />Создайте дискуссию на гите аддона<br />вроде он <a href="https://github.com/vlmaksime/weather.gismeteo">https://github.com/vlmaksime/weather.gismeteo</a><br />без самого скрипта трудно точно определить
            
                
            

            <div class="post-related">
                <div class='attachments'>
                    
                </div>
            </div>
        </td>
    </tr>
    </tbody>
</table>

            
        </div>
        <div>&nbsp;</div>
        
            <div class="text-center">
    



</div>

        

        

        
            
<a href="../../../accounts/register/index.html">Зарегистрироваться</a> или <a href="../../../accounts/login/index4750.html?next=/forum/topic/21119/">войдите</a> чтобы оставить сообщение.

        

        
    </div>

        </div>
    </div>

        </div>
    </div>
    <div class="row">
        <div class="col-md-6 footer">
            
                <p class="copyright">&copy; 2006-2022, Русскоязычное сообщество
                    Arch Linux.<br>Название и логотип Arch Linux &trade;
                    являются признанными торговыми
                    марками.<br>Linux &reg; &mdash; зарегистрированная торговая
                    марка Linus Torvalds и LMI.</p>
            
        </div>
    </div>
</div>

</body>

<!-- Mirrored from archlinux.org.ru/forum/topic/21119/ by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Sep 2022 04:11:53 GMT -->
</html>
